src/components/Datagrid/TooltipContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { Tooltip } from "@saleor/macaw-ui";
+import { Box } from "@saleor/macaw-ui/next";
+import React from "react";
+
+interface TooltipContainerProps {
+  clearTooltip: () => void;
+  title?: string;
+  bounds?: {
+    x: number;
+    y: number;
+    width: number;
+    height: number;
+  };
+}
+
+export const TooltipContainer: React.FC<TooltipContainerProps> = ({
```

Comment:
**suggestion**: can you add todo comment that this tooltip will be migrated to new one from the new Macaw?